src/main/java/com/catcher/core/dto/request/ScheduleListRequest.java

```diff
@@ -0,0 +1,21 @@
+package com.catcher.core.dto.request;
+
+import com.catcher.core.domain.entity.enums.SearchOption;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+
+@Getter
+@Setter
+public class ScheduleListRequest {
+    private Long participantFrom;
+    private Long participantTo;
+    private Long budgetFrom;
+    private Long budgetTo;
+    private String keyword;
+    private SearchOption keywordOption;
+    private String categoryName;
+    private LocalDateTime startAt;
```

Comment:
```suggestion
    @JsonFormat(pattern = "yyyy-MM-dd")
    private LocalDateTime startAt;
```
나중에 Date를 가져오는 부분은 JsonFormat 을 사용하는 것도 좋아보여요!